scripts/idr0044_zarr_segmentation_cluster.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+#
+# Copyright (c) 2020-2022 University of Dundee.
+#
+#   Redistribution and use in source and binary forms, with or without
+#   modification, are permitted provided that the following conditions
+#   are met:
+#
+#   Redistributions of source code must retain the above copyright notice,
+#   this list of conditions and the following disclaimer.
+#   Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#   THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+#   AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+#   IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+#   ARE DISCLAIMED.
+#   IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY
+#   DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY OR CONSEQUENTIAL DAMAGES
+#   (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+#   SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+#   HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+#   LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+#   OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
+#   DAMAGE.
+#
+# Version: 1.0
+#
+
+from dask.distributed import Client, LocalCluster
+
+import dask.array as da
+import dask_image.ndfilters
+import dask_image.ndmeasure
+
+import matplotlib.pyplot as plt
+import numpy
+
+import time
+
+# Dask array loaded from S3
+data = None
+
+
+# Load-binary
+def load_binary_from_s3(id, resolution='4'):
+    endpoint_url = 'https://uk1s3.embassy.ebi.ac.uk/'
+    root = 'idr/zarr/v0.1/%s.zarr/%s/' % (id, resolution)
+    return da.from_zarr(endpoint_url + root)
+
+
+# Segment-image
+def analyze(t, c, z):
+    plane = data[t, c, z, :, :]
+    smoothed_image = dask_image.ndfilters.gaussian_filter(plane, sigma=[1, 1])
+    threshold_value = 0.75 * da.max(smoothed_image).compute()
```

Comment:
```suggestion
    threshold_value = 0.33 * da.max(smoothed_image).compute()
```

with ``0.33`` I have much more recognizable shape on the segmented image, see screenshot below

![Screenshot 2022-04-08 at 14 59 50](https://user-images.githubusercontent.com/2478303/162473047-b518a343-99ff-45ac-ad56-7aef38d0a9b1.png)